zeppelin/executors/notebook_executor.py

```diff
@@ -56,6 +63,13 @@ def save_notebook(self, body):
                   self.output_path +
                   ' -- Please provide a valid absolute path.')
 
+    def get_version(self, body):
+        """Return correct version of Zeppelin file based on JSON format."""
+        if 'results' in body['paragraphs'][0]:
+            return '0.7.1'
+        else:
+            return '0.6.2'
```

Comment:
I think hard-coding the version numbers may not be the best approach. Consider what happens when v0.8 or later comes out. How do we maintain these?